src/vm/dllimport.cpp

```diff
@@ -5876,9 +5876,11 @@ bool         NDirect::s_fSecureLoadLibrarySupported = false;
 #ifdef FEATURE_PAL
 #define PLATFORM_SHARED_LIB_SUFFIX_W PAL_SHLIB_SUFFIX_W
 #define PLATFORM_SHARED_LIB_PREFIX_W PAL_SHLIB_PREFIX_W
+#define PLATFORM_PATH_DELIM W("/")
 #else // !FEATURE_PAL
 #define PLATFORM_SHARED_LIB_SUFFIX_W W(".dll")
 #define PLATFORM_SHARED_LIB_PREFIX_W W("")
+#define PLATFORM_PATH_DELIM W("\\")
```

Comment:
Can you use DIRECTORY_SEPARATOR_STR_W instead of define new one?

Reply:
Absolutely! I kept searching for 'delimiter', 'delim', etc and couldn't find anything. I will switch to that.